README.md

```diff
@@ -1,54 +1,91 @@
 
-Use the example of how we want to have an 'about' page on our site. So let's try to go there, let's just type localhost:3000/about.
+## Implementing a Static Route
 
-We get a routing error. Let's draw the route.
+Let's try this out in our application, I'm going to continue using the ```blog-flash``` application that I walked through in the Rails Basic lesson.
 
-Let's draw a simple route, like /about. Let's have that map to a StaticController and an about action (though don't create that controller). Reload the page. No more routing error, now it can't find the controller.
+To begin, startup the rails server and go to ```localhost:3000/about``` as you will see this throws a routing error: ```No route matches [GET] "/about"```, to fix this, stop the rails server by pressing ```control + c``` (anytime you make a routing change you need to restart the rails server)
 
-Let's make a new controller, no generators, with touch app/controllers/static_controller.rb.
+Now draw the route by opening the ```config/routes.rb``` file and add the following route inside of the ```draw``` block:
 
-Controllers need to inherit from ApplicationController and explain briefly the controller naming convention.
+```ruby
+get 'about', to: 'static#about'
+````
 
-Define our first controller action as a method, def about.
+Now start the rails server back up and go back to ```localhost:3000/about``` and click refresh, you should now see that the error message has changed, it's no longer complaining about not having a route, the error should now say: ```uninitialized constant StaticController```
 
-Let's hit refresh. Now it complains about a view.
+Let's fix this by creating a new controller for our static pages, you can do this manually by running the following command in the terminal:
 
-Controller actions are responsible for determining what the response is - responses are built by views.
+```touch app/controllers/static_controller.rb```
 
-For some reason we'll discuss briefly, rails is looking for a view in a very specific place. lets just make it happy and build our view in views/static/about.html.erb.
+This will create a blank controller file that we can use to map to the routing file. Since there are a number of methods built into the Rails controller system, you will also want the controller to inherit from the application controller. The new file should have code that looks like this:
 
-Let's just put some basic HTML there.
+```ruby
+class StaticController < ApplicationController
+end
+```
 
-Okay, now everything works.
+The standard naming convention for controllers is the name of the controller followed by the word ```controller```.
 
-Let's make this rendering process of choosing a view a little more explicit using the rails render method (equiv to sinatra's erb method if you're familair with it).
+If you refresh the browser now you will see a new error: ```The action 'about' could not be found for StaticController```
```

Comment:
also comment on how for changes to controllers, you *don't* need to restart the server